ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Block/Abstract.hs

```diff
@@ -250,7 +250,13 @@ succWithOrigin = withOrigin minBound succ
   Ouroboros Genesis window
 -------------------------------------------------------------------------------}
 
--- TODO docs/better place?
+-- | Size of the Genesis window, in number of slots.
+--
+-- This is the number of slots that the GDD Governor (Genesis Density
+-- Disconnection Governor -- see 'Ouroboros.Consensus.Genesis.Governor') will
+-- consider when deciding whether to disconnect from a peer. It has to be
+-- inferior or equal to the stability window and will often be equal to it. For
+-- instance, for Shelly-based eras on Cardano mainnet, this will be @3k/f@.
```

Comment:
```suggestion
-- instance, for Shelley-based eras, this will be @3k/f@ (one stability window).
```
(all Shelley-based eras, even non-mainnet, use the same formula `3k/f` for the stability window)